tests/unit/storage/data/external/ExternalClients.js

```diff
@@ -154,6 +164,45 @@ describe('external backend clients', () => {
                     });
             });
         });
+
+        if (backend.config.type !== 'azure') {
+            it(`${backend.name} objectPutTagging() then objectDeleteTagging() should set tags then delete it`, done => {
```

Comment:
To keep it simple,
`it(${backend.name} should set tags and then delete it, done => {});`